### ISSUE TITLE
The "Album art" folder fill with thousands of files 

### ISSUE THREAD

**Issue by Xara** [AzuraCast/AzuraCast#2564]
Hello, I hope that all of you are doing well and stay in safe places ...

**Using Docker installation method**
No

**Host Operating System**
Ubuntu 18.04 LTS

**Describe the bug**

Le bug is simple, each time when azuracast play a song, in album art is created new cover fille and old ones for same album are not erased witch lead to several Gb of disk space lost, especially for multi radios instance like mine.
**To Reproduce**
Check Album Art folder for each station,,,,

**Expected behavior**

**Relevant Logs**

**Screenshots**

**Device(s):**
 - Device:  
 - OS: 
 - Browser: 
 - Version: 

**Additional context**

**Comment by SlvrEagle23** [comment]
@Xara After looking through the relevant code, I can't seem to reproduce where this issue would be happening. Anywhere that album art is changed, it's either replacing the old file but with the same filename, or deleting the previous filename. Old album art is also deleted when the file itself is removed, during the normal processing steps.

Are you sharing your media across multiple stations? I would be curious to see about getting more information on this issue, but as it stands right now it doesn't seem to be happening at a large scale.

**Comment by Xara** [comment]
Hello, sorry with all this virus I was a bit away, it did me around 22G of trumb files in few days, I used cron to clean all stations once by day to avoid to go off space, now I have moved to new server, I will see how it act. Yes, I use cifs disks as media folder to most of stations, people or play from that common pool or they stream live. They have of course own folders too but none use private storage all run shared ones. Only problem that I have had faced with that config, is sometimes timeout on server boot, but it just restart and all is good again.
